lumicks/pylake/force_calibration/calibration_item.py

```diff
@@ -299,8 +304,19 @@ def _excluded_ranges(self):
         ]
 
     @property
-    def _fit_range(self):
-        """Return the spectral fit range used for the calibration"""
+    def _fit_range(self) -> tuple[float, float] | None:
+        """Return the actual spectral fit range used for the calibration"""
+        if "Adaptive fit range (min.) (Hz)" in self.data:
```

Comment:
Do you need to check also for `Adaptive fit range (max.) (Hz)` or you can assume it's always there if `min` is present?

Reply:
Currently, I'm using this purely as a differentiator between the two cases and would indeed expect that you would always have both. I mean, if we check both, then I would sooner raise if only one of them is present, because that condition doesn't make sense.